### ISSUE TITLE
Final Peer Review

### ISSUE THREAD

**Issue by zs366** [noriolg/ORIE4741_project#10]
Summary of the project:
This project aims to investigate the impacts of COVID on the Airbnb listings and predict the listing prices before and after COVID. They are using the datasets from Inside Airbnb which consist of rental information from April to September in 2019 and 2020. They explore the proximal gradient regression with Huber, Quadratic, and L1 loss, as well as Random Forest, XGBoost, and SVR models.

Three things I like:
1. The group has taken into account lots of features and it is interesting to see that room type is the most important feature for listing prices.
2. XGBoost is a powerful tool that uses gradient descent algorithms to minimize loss. It is great to see the group tried to predict using XGBoost!
3. The group provided lots of mathematical formulas that helped to understand the algorithms of their models.

Three areas for improvement:
1. Some features might be somehow correlated to each other, so the group should maybe address multicollinearity in the dataset.
2. It could be better if they explain in detail how they handled/imputed missing values for their project.
3. The analysis and conclusion parts could be more detailed and through with more explanations. 

Overall, it is a very interesting project, great work!
